api/v1beta1/manila_webhook.go

```diff
@@ -44,6 +44,7 @@ type ManilaDefaults struct {
 	ShareContainerImageURL     string
 	DBPurgeAge                 int
 	DBPurgeSchedule            string
+	APITimeout int
```

Comment:
nit: Please align `int` with the `string` of the previous line

Reply:
Done